src/quicktype-core/language/Python.ts

```diff
@@ -417,6 +419,10 @@ export class PythonRenderer extends ConvenienceRenderer {
                 if (t.getProperties().size === 0) {
                     this.emitLine("pass");
                 } else {
+                    const sort = (v: ClassProperty) => (
+                        Boolean(v.type instanceof UnionType && Boolean(nullableFromUnion(v.type))) ? 1 : 0
+                    );
+                    t.setProperties(mapSortBy(t.getProperties(), sort), undefined, false);
```

Comment:
Again, not the way to do this.  A much nicer way and more general way to do it would be to generalize `ConvenienceRenderer.forEachClassProperty` by adding a `protected` method `sortClassProperties`, which, in `ConvenienceRenderer` would either just return the properties in their natural order or, if `alphabetizeProperties` is set, in alphabetical order.  Then `PythonRenderer` could overwrite it to use its own ordering.

Reply:
implemented in 762d2fc